ccd-definition/CaseEventToFields/otherAllocationDecision.json

```diff
@@ -2,43 +2,43 @@
   {
   "LiveFrom": "01/01/2017",
   "CaseTypeID": "CARE_SUPERVISION_EPO",
-  "CaseEventID": "otherProposalDecision",
+  "CaseEventID": "otherAllocationDecision",
   "CaseFieldID": "allocationDecision_Label",
   "PageFieldDisplayOrder": 1,
   "DisplayContext": "READONLY",
   "PageID": 1,
   "PageDisplayOrder": 1,
   "PageColumnNumber": 1,
-  "FieldShowCondition": "allocationProposal!=\"\""
+  "FieldShowCondition": "allocationProposalPresent!=\"No\""
```

Comment:
I see you changed the variable to be more happy path friends which is nice. Could you maybe change the condition itself to equals? Would be super clear, allocationProposalPresent=\"Yes?
What do you think? :)

Reply:
discussed with you that i did it like this due to ccd constraint of indenting labels with grey